csrc/mmdeploy/preprocess/transform/pad.cpp

```diff
@@ -82,9 +82,9 @@ class Pad : public Transform {
           data["pad_fixed_size"].push_back(pad_h);
           data["pad_fixed_size"].push_back(pad_w);
         } else {
-          padding = {0, 0, size_[1] - width, size_[0] - height};
-          data["pad_fixed_size"].push_back(size_[0]);
+          padding = {0, 0, size_[0] - width, size_[1] - height};
           data["pad_fixed_size"].push_back(size_[1]);
+          data["pad_fixed_size"].push_back(size_[0]);
```

Comment:
https://github.com/open-mmlab/mmcv/blob/e322848e3a466f2b1f5b2e1a9cb5552342db1e93/mmcv/transforms/processing.py#L386

pad_fixed_size seems to be in (w,h) format, although it is not used elsewhere

Reply:
That's weird. In mmcv's document, pad_shape is formed with (height, width) which should be self.size [here](https://github.com/open-mmlab/mmcv/blob/e322848e3a466f2b1f5b2e1a9cb5552342db1e93/mmcv/transforms/processing.py#L386).

Reply:
Check this.
> https://mmcv.readthedocs.io/en/2.x/understand_mmcv/data_transform.html
> 
> ![image](https://user-images.githubusercontent.com/41138331/237061112-93beee43-ef1e-4c2b-960e-1a5cac33ca38.png)